src/app/backend/replicasetcommon.go

```diff
@@ -100,3 +114,78 @@ func getReplicaSetPodInfo(replicaSet *api.ReplicationController, pods []api.Pod)
 
 	return result
 }
+
+// Returns Pod Events of type warning from given list of pods
+// TODO(floreks) : use some set to get only unique error messages
```

Comment:
I've actually wanted to import `Set` instead of writing workaround function. Godeps are hard so for now I've just left this function. :)